docs/quickstart/quickstart.md

```diff
@@ -177,6 +177,72 @@ tanzu plugin upgrade <plugin> [--target <target>]
 This command will update the specified plugin to the recommendedVersion
 associated with this plugin's entry found in the plugin repository.
 
+### Logging into Tanzu Application SaaS Platform
+
+To log in to the Tanzu Application SaaS platform and create a context of type tanzu, use the following command:
+
+```console
+tanzu login
+```
+
+This command will log in to the Tanzu Application SaaS platform and creates a context associated
```

Comment:
The references to "Tanzu Application SaaS platform" here and a few other locations should be update Tanzu Application Platform

Reply:
Thank you! Updated the docs tos use `Tanzu Application Platform`